src/_pytest/terminal.py

```diff
@@ -1249,9 +1263,14 @@ def _build_collect_only_summary_stats_line(
         return parts, main_color
 
 
-def _get_pos(config: Config, rep: BaseReport):
+def _get_node_id_with_markup(tw: TerminalWriter, config: Config, rep: BaseReport):
     nodeid = config.cwd_relative_nodeid(rep.nodeid)
-    return nodeid
+    path, *parts = nodeid.split("::")
+    if parts:
+        parts_markup = tw.markup("::".join(parts), white=True, bold=True)
```

Comment:
Hmm I'm not sure but wouldn't `white=True` be a problem on white terminal backgrounds? Why is it needed?

Reply:
Right, this was probably left-over from testing, removed. 👍